src/control/server/mgmt_svc.go

```diff
@@ -42,7 +43,11 @@ import (
 	"github.com/daos-stack/daos/src/control/system"
 )
 
-const instanceUpdateDelay = 500 * time.Millisecond
+const (
+	instanceUpdateDelay = 500 * time.Millisecond
+	minScmBytes         = 16 * humanize.MiByte // per VOS target
+	minNvmeBytes        = 1 * humanize.GByte   // per VOS target
```

Comment:
minNvmeBytes should actually be 1 GiB to match the define in src/bio/bio_xstream.c (which has a misleading comment, sigh):
```C
/* SPDK blob parameters */
#define DAOS_BS_CLUSTER_SZ      (1ULL << 30)    /* 1GB */
```

Reply:
will fix the comment as well, good catch